packages/components/psammead-headings/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Psammead Headings Changelog
+
+| Version | Description |
+|---------|-------------|
+| 0.1.0   | [PR#79](https://github.com/BBC-News/psammead/pull/79) Create initial package, pulled in from [simorgh](https://github.com/BBC-News/simorgh). |
```

Comment:
Shouldn't this be 1.0.0? I appreciate we may still make tweaks to the component as and when we discover complexities, but wouldn't have thought we risk having to publish a v2 any time soon.

Reply:
@ChrisBAshton This issue that's just been opened against this component: https://github.com/BBC-News/psammead/issues/88  is removing a prop. This would require a major bump in the version number.

Reply:
Re: it being 0.1.0 not 1.0.0 thats just the convention we have used for the other packages

Reply:
Ha, fair point! I'd argue it's a minor (1.1) as continuing to supply the prop wouldn't _break_ compilation. But maybe we _should_ keep everything as 0.1.0 until all components are moved, then mass-upgrade them all to v1 when we're happy...

Reply:
I agree that it'd be good to keep it at 0.1.0 to start with, since there can be breaking changes before version 1.0.0 is released.